HW_2/task_1.py

```diff
@@ -0,0 +1,24 @@
+"""
+Напишите программу, которая принимает на вход вещественное число и показывает сумму его цифр.
+
+Ввод: значение типа <float>
+Вывод: значение типа <int>
+
+Примеры:
+6782.0
+23
+
+0.56
+11
+"""
+N = input('Введите число N = ')
+sum = 0
+for i in str(N):
```

Comment:
```suggestion
for i in N:
```
Эта переменная уже типа `str`.